recipes/im-pipelines-utils-rdkit/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "im-pipelines-utils-rdkit" %}
```

Comment:
Is the prefix im- really needed?

Reply:
It's not strictly needed but I think it makes sense to have the same name as on PyPi: https://pypi.org/project/im-pipelines-utils-rdkit/

Reply:
@bgruening, is this okay? If so, can this be merged?